src/accessor/mod.rs

```diff
@@ -138,6 +140,22 @@ impl<'a> Accessor<'a> {
         self.json.component_type.unwrap().0
     }
 
+    /// Returns the extension values map
+    #[cfg(feature = "extensions")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "extensions")))]
+    pub fn extensions(&self) -> Option<&Map<String, Value>> {
+        let ext = self.json.extensions.as_ref()?;
+        Some(&ext.others)
+    }
+
+    /// Return a value for a given extension name
+    #[cfg(feature = "extensions")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "extensions")))]
+    pub fn get_extension_value(&self, ext_name: &str) -> Option<&Value> {
```

Comment:
Could we rename this simply `extension`? Did you think that was too similar?

Reply:
yeah I felt like the s wasn't dissimilar enough. I could remove the `get` from the name so it'd just be `extension_value()`?

Reply:
Sounds good to me. :+1: